### PR TITLE
AT_001_000 | <Sing in>

### PR DESCRIPTION
**Pull request by StanislavLiv**
TC_001_000 | <Sing in>
https://trello.com/c/rJhiwdf4/329-tc001000-sing-in

AT_001_000 | <Sing in>
https://trello.com/c/wwADghUF/332-at001000-sing-in

### CODE REVIEW

File: src/test/java/runner/StasSingInTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+public class StasSingInTest extends BaseTest {
+
+    private WebDriver driver;
+
+    @BeforeMethod
+    @Override
+    protected void beforeMethod() {
+        driver = BaseUtils.createDriver();
+    }
```
Comment (RusAli) [inline src/test/java/runner/StasSingInTest.java:18]: U extend this method from BaseTest class

Comment (StanislavLiv) [reply src/test/java/runner/StasSingInTest.java:18]: I tried to explain my test logic above. I don't need to extends from BaseTest

File: src/test/java/runner/StasSingInTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    @AfterMethod
+    @Override
+    protected void afterMethod() {
+        driver.quit();
+    }
```
[the remark this answers is not in the document]
Comment (StanislavLiv) [reply src/test/java/runner/StasSingInTest.java:24]: Hi Rus. Engineers have local properties (name and password), and then we are testing Jenkins UI. My test is testing Log In page before we use local properties.
I am going to refactor it later and will delete annotations Override

[the remark this answers is not in the document]
Comment (RusAli) [reply src/test/java/runner/StasSingInTest.java:24]: got it

**Review by RusAli — APPROVED** [review]

**Pull request closed without merging**